node/src/reactor/validator/tests.rs

```diff
@@ -235,29 +211,37 @@ async fn run_equivocator_network() {
     keys.push(alice_sk);
 
     let mut chain = TestChain::new_with_keys(&mut rng, keys, stakes);
+    let protocol_config = (&*chain.chainspec).into();
 
     let mut net = chain
         .create_initialized_network(&mut rng)
         .await
         .expect("network initialization failed");
 
+    info!("Waiting for Era 0 to end");
+    net.settle_on(&mut rng, is_in_era(EraId(1)), Duration::from_secs(600))
+        .await;
+
     info!("Waiting for Era 1 to end");
-    net.settle_on(&mut rng, is_in_era(1), Duration::from_secs(600))
+    net.settle_on(&mut rng, is_in_era(EraId(2)), Duration::from_secs(90))
```

Comment:
This needs to stay at 600 for slower test machines.

Reply:
I didn't change this: The duration for the first era (above) is still 600 seconds, and 90 for all the others.